scheduler/src/cook/config.clj

```diff
@@ -424,7 +419,8 @@
                    (when (:min-utilization-threshold rebalancer)
                      (log/warn "The :min-utilization-threshold configuration field is no longer used"))
                    (merge {:interval-seconds 300
-                           :dru-scale 1.0}
+                           :dru-scale 1.0
+                           :rebalancer-pools ".*"}
```

Comment:
Maybe just `:pool-regex` since you're already in the `:rebalancer` section?

Reply:
Done